spec/factories/items.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :item do
+    name "Gift1"
```

Comment:
I can change this name to whatever, it's "Gift1" just because I didn't have any better idea :) I would leave it this way - in next issue I'll add Faker to those tests anyway